src/main/java/io/vertx/cassandra/CassandraClient.java

```diff
@@ -76,7 +76,29 @@ default CassandraClient connect(Handler<AsyncResult<Void>> connectHandler) {
    * @return current Cassandra client instance
    */
   @Fluent
-  CassandraClient execute(String query, Handler<AsyncResult<ResultSet>> resultHandler);
+  default CassandraClient execute(String query, Handler<AsyncResult<ResultSet>> resultHandler){
+    return execute(ExecutableQuery.fromString(query), resultHandler);
+  }
+
+  /**
+   * Execute the query and provide a handler for consuming results
```

Comment:
Minor: Javadoc is invalid if first sentence does not end with a period. Javadoc uses the first sentence in the class summary table.